.github/workflows/ci.yml

```diff
@@ -43,6 +43,10 @@ jobs:
         with:
           credentials: https://user:${{ secrets.GITHUB_TOKEN }}@github.com/
```

Comment:
Not fully sure, but I think, the "de-vri-es" could be removed: It's for accessing private repos ...

Reply:
at least. we should give the option to access private repos, when you generate a repo from the template it's not always public